docker-compose-sierra-leone.yml

```diff
@@ -0,0 +1,72 @@
+version: '3'
+services:
+  postgres-db:
+    container_name: postgres-database-sierra-leone
+    image: pgracio/dhis2-db:2.27-sierra-leone
+    environment:
+      POSTGRES_USER: dhis
+      POSTGRES_DB: dhis
+      POSTGRES_PASSWORD: dhis
+      POSTGRES_DB_TARGET: dhis-target
+      PG_DATA: /var/lib/postgresql/data/pgdata:z
+    volumes:
+      - "/data/postgres-sierra-leone:/var/lib/postgresql/data/pgdata"
```

Comment:
Can we rather use named volumes? It's not ideal to have docker creating directories on the host machine and bind mounting them. It also introduces the possibility of conflicts between containers.